rxandroidble/src/main/java/com/polidea/rxandroidble2/LogOptions.java

```diff
@@ -0,0 +1,169 @@
+package com.polidea.rxandroidble2;
+
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+
+public class LogOptions {
+
+    @Nullable
+    private Integer logLevel;
+    @Nullable
+    private Integer macAddressLogSetting;
+    @Nullable
+    private Integer uuidLogSetting;
+    @Nullable
+    private Boolean shouldLogAttributeValues;
+    @Nullable
+    private Logger logger;
+
+    private LogOptions(@Nullable Integer logLevel, @Nullable Integer macAddressLogSetting, @Nullable Integer uuidLogSetting,
```

Comment:
I guess there should be documentation somewhere about default behaviour,

Reply:
Still not present?

Reply:
I am not sure if it would be visible on the private constructor. Added some docs
`com/polidea/rxandroidble2/LogOptions.java:11`
`com/polidea/rxandroidble2/LogOptions.java:75`
`com.polidea.rxandroidble2.RxBleClient#updateLogOptions`
Do you think that here they also should be present?

Reply:
Finally understood after an offline conversation. Adressed [here](https://github.com/Polidea/RxAndroidBle/pull/551/commits/747b5612a17d59230d50f810df6c8967e0380462)